src/Contact/Contact.test.js

```diff
@@ -0,0 +1,20 @@
+import { screen, render } from '@testing-library/react'
+import Contact from './Contact'
+import { samplePark } from '../samplePark'
+import '@testing-library/jest-dom'
+
+describe('Contact', () => {
+
+  beforeEach(() => {
+    render(
+      <Contact
+        currentPark={samplePark}
```

Comment:
We're going to want to put this sample Park in a separate file and just import it. It's massive! Can do that in the next branch